virttest/qemu_vm.py

```diff
@@ -4784,6 +4786,10 @@ def process_info_block(self, blocks_info):
         """
         block_list = []
         block_entry = []
+
+        if not isinstance(blocks_info,str):
```

Comment:
```suggestion
        if not isinstance(blocks_info, (str, bytes)):
```
I think that we should consider other types that support `isinstance()` calling.